src/stratum/stratumConnection.js

```diff
@@ -100,13 +100,13 @@ export class StratumConnection {
     logger.info(`${this.walletId} stratum.open: ${this.uri}`)
     const parsed = parse(this.uri)
     if (
-      (parsed.scheme !== 'electrum' &&
-        parsed.scheme !== 'electrums' &&
-        parsed.scheme !== 'electrumwss') ||
+      (parsed.scheme !== 'electrum' && parsed.scheme !== 'electrums') ||
```

Comment:
Oops, accidentally un-did your earlier change. Put `electrumwss` this back in this list, like it was.

Reply:
@swansontec Should `electrumws` also be in this list?

Reply:
Yes, `electrums` is fine in this list, since this routine can handle that case. The problem with `electrums` is elsewhere.